app/src/main/assets/native/ExoPlayerPlugin.js

```diff
@@ -163,17 +163,15 @@ export class ExoPlayerPlugin {
     }
 
     async getDeviceProfile() {
-        var profile = {};
-        profile.Name = 'ExoPlayer Stub';
-        profile.MaxStreamingBitrate = 100000000;
-        profile.MaxStaticBitrate = 100000000;
-        profile.MusicStreamingTranscodingBitrate = 320000;
-
-        profile.DirectPlayProfiles = [];
-        profile.CodecProfiles = [];
-        profile.SubtitleProfiles = [];
-        profile.TranscodingProfiles = [];
-
-        return profile;
+        return {
+            'Name': 'ExoPlayer Stub',
```

Comment:
Object keys in JS don't need to be quoted:

```suggestion
            Name: 'ExoPlayer Stub',
```

(same for the keys in the DirectPlayProfiles array)

Reply:
Copied from JMP, oops. Gonna fix.